substrate/bin/node/inspect/Cargo.toml

```diff
@@ -1,13 +1,12 @@
 [package]
-name = "node-inspect"
+name = "staging-node-inspect"
```

Comment:
We can revert the renames of the crates that are jot getting published?

Reply:
Even if not publishing, I don't think it's a good idea to use names that are taken on crates io. Could lead to confusion and people trying to fetch them.

Reply:
https://crates.io/crates/node-template is for example also taken. I don't see any problems in using these names as long as we don't publish them.